### PR TITLE
Bootstrap motherfuckers!

### PR DESCRIPTION
**Pull request by jonathan-ostrander**
Can you spare 100kb pls?

### CODE REVIEW

File: list.yaml [hunk 1]
```diff
@@ -190,6 +190,6 @@
 - title: Escape the Room
   desc: Have people or admins be able to create rooms that can be solved by people through IRC-like chat
   details:
-    -Make it so that it's necessary for N people to comment in certain ways to solve the room
-    -Make rooms time based so they will come back at certain times or build their own tools to solve rooms
-    -Allow advertisers to build their own rooms and have users solve puzzles based around rooms to get rewards/discounts
+    - Make it so that it's necessary for N people to comment in certain ways to solve the room
```
Comment (SharangP) [inline list.yaml:193]: @OstrichProjects i already fixed this pull master and rebase your branch, or just undo these changes before pulling master into your branch:

`git checkout your_branch`
`git fetch && git rebase origin/master`

then fix your merge conflicts
`git add list.yaml`
then `git rebase --continue`

Comment (jonathan-ostrander) [inline list.yaml:193]: Too hard, just bootstrap it with dildos

Jonathan Ostrander
(516) 640-2963
jonathanost@gmail.com

File: list.yaml [hunk 2]
```diff
@@ -200,3 +202,7 @@
     - online portal to view stored items
     - dependant on "network" i.e. shipping being cheap
 
+=======
```
Comment (SharangP) [inline list.yaml]: wtf is this shit @OstrichProjects you rebased incorrectly probably. just delete any changes to this file and ill approve

**Comment by jonathan-ostrander** [comment]
Fixed bb

File: list.yaml (hunk 2 above)
Comment (jonathan-ostrander) [inline list.yaml]: windows brah

Comment (SharangP) [inline list.yaml]: idc i merged.

On Tue, Nov 3, 2015 at 3:24 PM, Jonathan Ostrander <notifications@github.com

> wrote:
> 
> In list.yaml
> https://github.com/SharangP/ideas/pull/3#discussion_r43801849:
> 
> > @@ -200,3 +202,7 @@
> >      - online portal to view stored items
> >      - dependant on "network" i.e. shipping being cheap
> > 
> > +=======
> 
> windows brah
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/SharangP/ideas/pull/3/files#r43801849.

## 

Sharang Phadke
http://sharangphadke.me

**Comment by choochootrain** [comment]
:shit:

**Comment by jonathan-ostrander** [comment]
😹😹😹

Jonathan Ostrander
(516) 640-2963
jonathanost@gmail.com
